rekor/rekor.go

```diff
@@ -78,9 +78,9 @@ func New(rekorServer string) (RekorClient, error) {
 }
 
 func (r *wrappedRekorClient) StoreArtifact(artifactBytes, pubkeyBytes []byte) (*entries.CreateLogEntryCreated, error) {
-	entry, err := types.NewProposedEntry(context.Background(), "dsse", "0.0.1", types.ArtifactProperties{
+	entry, err := types.NewProposedEntry(context.Background(), "intoto", "0.0.2", types.ArtifactProperties{
 		ArtifactBytes:  artifactBytes,
-		PublicKeyBytes: pubkeyBytes,
+		PublicKeyBytes: [][]byte{pubkeyBytes},
```

Comment:
Newer versions of rekor changed the `PublicKeyBytes` from `[]byte` to `[][]byte`. I didn't want to break any compatibility so I kept the function with `[]byte` and just cast it to `[][]byte`. I don't think this causes any issues but wanted to point it out.